### PR TITLE
Use v4 functions for login and logout in gsctl

### PR DESCRIPTION
**Pull request by marians**
For https://github.com/giantswarm/giantswarm/issues/1524 https://github.com/giantswarm/gsctl/issues/236

Depends on  https://github.com/giantswarm/gsclientgen/pull/30

TODO:

- Fix tests
- Change gsclientgen back to `master`

### CODE REVIEW

File: Gopkg.toml [hunk 1]
```diff
@@ -38,16 +38,12 @@
...
-  version = "0.7.1"
-
 [[constraint]]
   name = "github.com/giantswarm/columnize"
   version = "=2.0.2"
 
 [[constraint]]
-  branch = "master"
+  branch = "v4-login"
   name = "github.com/giantswarm/gsclientgen"
```
Comment (pipo02mix) [inline Gopkg.toml]: should it be changed back to master after test?

Comment (marians) [reply Gopkg.toml]: Yes, as it says in the main comment.

**Comment by marians** [comment]
Sorry, I asked for a review too early. Seems I have to fix some unit test(s) first.

**Comment by marians** [comment]
There is no point in doing this with the client generated by swagger-codegen. Reason: We have no access to the response body in the case of HTTP response status >= 400. And we need the response to know what went wrong in logins.

Closing. Will redo one we are generating the client using go-swagger.

**Pull request closed without merging**